### PR TITLE
tries installing Python and running a sample script

### PR DESCRIPTION
**Pull request by maneesha**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ before_install:
   - sudo add-apt-repository -y "ppa:ubuntugis/ppa"
   - sudo add-apt-repository -y "ppa:cran/travis"
   - travis_apt_get_update
-  - sudo apt-get install -y --no-install-recommends build-essential gcc g++ libblas-dev liblapack-dev libncurses5-dev libreadline-dev libjpeg-dev libpcre3-dev libpng-dev zlib1g-dev libbz2-dev liblzma-dev libicu-dev cdbs qpdf texinfo libssh2-1-dev gfortran jq
+  - sudo apt-get install -y --no-install-recommends build-essential gcc g++ libblas-dev liblapack-dev libncurses5-dev libreadline-dev libjpeg-dev libpcre3-dev libpng-dev zlib1g-dev libbz2-dev liblzma-dev libicu-dev cdbs qpdf texinfo libssh2-1-dev gfortran jq python3.7 python3-pip
```
Comment (fmichonneau) [inline .travis.yml]: it looks like the highest version of python available for Ubuntu Xenial is python3.5 (https://packages.ubuntu.com/xenial/python3.5). Would this work for you?

Comment (maneesha) [reply .travis.yml]: Yes, I just put in a new commit.  How were you able to tell that was the problem?

Comment (fmichonneau) [reply .travis.yml]: in the logs of the failed travis build: https://travis-ci.com/carpentries/amy-feeds/builds/128943671#L245

File: .travis.yml [hunk 2]
```diff
@@ -45,6 +45,7 @@ before_install:
   - rm /tmp/pandoc-2.2-1-amd64.deb
   - R -q -e "install.packages(c('countrycode','maps','readr','dplyr','tidyr','plotly','ggplot2','htmlwidgets','svglite'))"
   - Rscript -e 'sessionInfo()'
+  - python -m pip3 install numpy scipy matplotlib ipython jupyter pandas
```
Comment (fmichonneau) [inline .travis.yml]: at least locally, I do `pip3 install ... --user` (or `sudo pip3 install ...`). It looks like with the `python -m` syntax, here it should be `python3 -m pip install ...`

Comment (fmichonneau) [reply .travis.yml]: see log here: https://travis-ci.com/carpentries/amy-feeds/builds/128949863#L6124

**Comment by maneesha** [comment]
Testing `after_success` seems to work.  See [travis log](https://travis-ci.com/carpentries/amy-feeds/builds/129128794#L6394).  Final PR to remove the test line.  @fmichonneau  I think we should be good to merge now.

**Pull request merged**